packages/lynxfetch/lynxfetch.pacscript

```diff
@@ -0,0 +1,22 @@
+pkgname="lynxfetch"
+arch=("amd64")
+pkgver="1.0.6"
+source=("https://gitlab.com/${pkgname}/${pkgname}/-/archive/${pkgver}/${pkgname}-${pkgver}.tar.gz")
+makedepends=("libdbus-1-dev" "pkg-config")
+pacdeps=("rust-bin")
+pkgdesc="Super fast, versatile, and customizable command-line fetcher"
+sha512sums=("29ecff0bf5382c5595ae8fee85970ba82d212056151cc304b602a1d4fd726bca4733b2d96dfd36b6d2c61788bf526b6b5b4e4296d28e4791af5cc7153a808629")
+maintainer=("CELESTIFYX Team <celestifyx@gmail.com>")
+external_connection=true
+
+build() {
+  cd "${pkgname}-${pkgver}"
+
+  cargo build --release
```

Comment:
```suggestion
  cargo build -j"${NCPU}" --release --locked
```

We usually follow https://github.com/pacstall/pacstall/wiki/Different-Package-Build-Systems#cargo for a blueprint on building rust packages!